docs/README.md

```diff
@@ -150,7 +150,7 @@ The following commands are used to bootstrap CodeChecker on Ubuntu 20.04 LTS:
 # NOTE: clang or clang-tidy can be any sufficiently fresh version, and need not
 #       come from package manager!
 sudo apt-get install clang clang-tidy build-essential curl doxygen gcc-multilib \
-      git python3-virtualenv python3-dev
+      git python3-dev python3-venv
```

Comment:
@gyorb The `python3-virtualenv` is not needed anymore?

Reply:
It should not be needed. [python3-virtualenv](https://virtualenv.pypa.io/en/latest/) can be used to create virtualenvs but some of its features were implemented in the standard package inside the python interpreter which is packaged into a separate deb package on ubuntu (python3-venv). Not all the features are implemented there but we just use the virtualenv creation.

Some limitations of python3-venv compared to virtualenv mentioned [here](https://virtualenv.pypa.io/en/latest/):
 - is slower (by not having the app-data seed method),
 - is not as extendable,
 - cannot create virtual environments for arbitrarily installed python versions (and automatically discover these),
 - is not upgrade-able via pip,
 - does not have as rich programmatic API (describe virtual environments without creating them).

 I do not think the codechecker environment depends on these extra features.